Shawntru/Rotten-Tomatillos
# Add Router feature to components

## Is this a fix or a feature?
Feature

## What is the change?
Refactoring a project with Router. 
MoviePreview was change to a class.
Removed button functionalities to come back to homepage and to a single movie page were replaced with Links.
Correct testing for using Router

## What does it fix?
SRP, readability and better organization of the components
Faster user experience through Router
Adds ability to use navigation and bookmark URL addresses 

## Where should the reviewer start?
App.js line 6
MovieCard.js line 3
MoviePreview.js line 1
Movies.js line 14
index.js line 9
And all related testing for each listed component 

## How should this be tested?
Running npm start would display a fully functional website where User can click on any movie and taken to a movie page. By using a closing button or browser arrows, the User can come back to the previous page
Ensure all testing passes using `npm test`

## Relevant Ticket(s)
#27 #26
